### DISCUSSION TITLE
Old Shimmer and Assembly Missing

### DISCUSSION THREAD

**Discussion by wireless90** [iotaledger/firefly#3355] [Help & Support]
Like others have pointed out, they are missing from my wallet. I spammed balance finder to no avail. Perhaps can we have a list of reasons why this is happening and how to solve it?

**Comment by MiKeF05** [comment]
Which Firefly version do you use? The most likely reason is that Firefly can´t fetch the information because the request is blocked. Do you have any firewall or filter active? Maybe you use a VPN?

Please be aware that a new feature has been implemented in Firefly. With the wallet switcher you can switch between your wallets. And all balances are linked to the corresponding wallet. So maybe you need to choose the correct wallet with which you have staked?
